rootfs/usr/local/bin/startup

```diff
@@ -13,6 +13,7 @@ fi
 
 # Set file config for nginx and php
 sed -i "s/<FLARUM_PORT>/${FLARUM_PORT}/g" /etc/nginx/nginx.conf
+sed -i "s/<FLARUM_PORT>/${FLARUM_PUBLIC_PATH}/g" /etc/nginx/nginx.conf
```

Comment:
`sed -i "s/<FLARUM_PUBLIC_PATH>/${FLARUM_PUBLIC_PATH}/g" /etc/nginx/nginx.conf` ?

Reply:
The reason I put vertical slashes is that the path can contain forward slashes, which messes with sed

Reply:
I think is <FLARUM_PUBLIC_PATH> instead <FLARUM_PORT>
I can't merge this.

Reply:
Yeah I updated this in  da00260f7c81af677736159418580e4ac062316f